electrasmart/client.py

```diff
@@ -204,8 +204,7 @@ def _modify_oper_and_send_command(self):
         )
 
     def modify_oper(
-        self, *, ac_mode=None, fan_speed=None, temperature=None, ac_stsrc="WI-FI"
-    ):
+        self, *, ac_mode=None, fan_speed=None, temperature=None, ac_stsrc="WI-FI", permissive_args=False, **kwargs):
```

Comment:
Why do `permissive_args=False, **kwargs`? why not add the explicit parameters for the different supported modes?
Let's try to keep this consistent across all params.
Could you share an example of how the parameters are supposed to be used? We can then figure this out together, if you'd like.

Reply:
Permissive_Argus will allow inputting any variable name, while false it will do input validation on names.
It could be nice for quick fixes but that's maybe a bad habit